shell_scripts/trust_your_luck.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+num=$(( $RANDOM % 4 ))
+sleep 2 
+
+echo "Player one enter your name."
+read player1
+sleep 1
+echo "$player1  choose a number from 1-3."
+read player1num
+
+sleep 1
+if [[ $player1num > 3 ]]; then
+echo "So this is what happened $player1 its either you picked a number greater than 3 or you pick an alphabet so stop  playing my guy you trying to deceive me or what?"
+else
+
+echo "Player two what is your name?"
+read player2
+
+sleep 1
+
+echo "$player2 choose a number from 1-3. "
+read player2num
+sleep 1
```

Comment:
in this case lines 17 to 24 is a repetition of lines 6 to 12,
we can clean this up using a function like `get_user_inputs`